Alcatraz/Installers/ATZPluginInstaller.m

```diff
@@ -71,6 +75,46 @@ - (NSString *)pathForInstalledPackage:(ATZPackage *)package {
                                        stringByAppendingString:package.extension];
 }
 
+- (BOOL)isPackageBlacklisted:(ATZPackage *)package {
+    if (![self isPackageInstalled:package]) {
+        return NO;
+    }
+
+    NSString *pluginManagerKey = [self xcodeDefaultsNonApplePluginsKey];
```

Comment:
Hmm. Are there any plan do switch anytime soon?
Reading the Xcode version from outside the plugin is fairly straightforward, but this gets more complicated and fragile if we're to handle multiple installations of Xcode, as we'll need to rely on `xcrun`/`xcode-select`. I'd be tempted to keep using `[NSBundle mainBundle]` until we can't anymore.